From ad8cc55abae7bb0e1494a03947f95c63813c83ce Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Tue, 6 Oct 2020 16:39:16 +0200
Subject: [PATCH 12/26] dnsmasq: turn the base uci section into a named one

Handling this section from opensync is complicated if it is unnamed.

Signed-off-by: John Crispin <john@phrozen.org>
---
 package/network/services/dnsmasq/files/dhcp.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/network/services/dnsmasq/files/dhcp.conf b/package/network/services/dnsmasq/files/dhcp.conf
index 19052e2333..3e54bac911 100644
--- a/package/network/services/dnsmasq/files/dhcp.conf
+++ b/package/network/services/dnsmasq/files/dhcp.conf
@@ -1,4 +1,4 @@
-config dnsmasq
+config dnsmasq dnsmasq
 	option domainneeded	1
 	option boguspriv	1
 	option filterwin2k	0  # enable for dial on demand
-- 
2.25.1

